scripts/get_bundle_test_path.py

```diff
@@ -0,0 +1,33 @@
+# Get bundle test path for specific release
+import re
+import sys
+import os
+
+
+def get_bundle_test_path_from_release() -> None:
+    if len(sys.argv) > 1:
```

Comment:
To improve readability could we have raise exception in this manner instead of wrapping the whole code in `if` clause?
```
if len(sys.argv) == 0:
  raise Exception("No release given as input.")
```

Reply:
Implementing but with `<= 1`